xml/deploy_cephadm.xml

```diff
@@ -726,11 +726,28 @@ o- time_server ................................................ [enabled]
     <procedure>
      <step>
       <para>
-       Create a local container registry accessible but outside of the &ceph;
-       cluster, for example:
+       Create a local container registry outside &ceph; cluster nodes that is
+       accessible from cluster nodes, for example:
```

Comment:
i know....1 sentence is not enough
```suggestion
       Create a local container registry instead of the default public one. Create it
       outside of &ceph; cluster nodes. The local registry needs to be accessible
       from cluster nodes. Use a command similar to the following:
```

Reply:
Yeah that's better @tbazant - apply that :)